runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordState.java

```diff
@@ -209,4 +209,11 @@ void checkForTimeout()
 		final Instant now = Instant.now();
 		events.removeIf(event -> event.getType().isShouldTimeout() && now.isAfter(event.getUpdated().plus(actionTimeout)));
 	}
-}
+
+	public void checkForMenuTimeout() {
```

Comment:
You'll want to fix your checkstyle. [The wiki has a guide on that](https://github.com/runelite/runelite/wiki/Checkstyle-with-IntelliJ-IDEA)